app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="login_section">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email, class:"email_label"  %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Username" %>
```

Comment:
- [ ] As requested by the previous reviewer, change `username` to `email` for the placeholder text
![image](https://user-images.githubusercontent.com/81932478/216368553-84521d52-5481-48c4-acaa-3550509970ae.png)